### DISCUSSION TITLE
Documentation Archives

### DISCUSSION THREAD

**Discussion by dremin** [twilio-labs/paste#2735] [Ideas]
I'm not able to find documentation for historical versions of Paste. This is making it fairly frustrating/difficult to develop for Flex UI 2.0 because it is currently using an older version of Paste. I suspect other consumers of Paste are also not consistently on the latest release. Would it be possible to provide documentation archives like other UI frameworks provide, so that we can view accurate documentation for the version we are developing with? Thanks!

**Comment by nkrantz** [comment]
Hi @dremin - let me know if these help or if there's anything else we can help you find.

[- Paste core changelog](https://paste.twilio.design/core/changelog)
[- Paste core upgrade guide](https://paste.twilio.design/core/upgrade-guide)

**Comment by TheSisb** [comment]
Hi @dremin,

I can totally understand how this would be frustrating. This is something we're exploring a solution for and we'll report back when we have something more concrete. Thank you for the feedback.

**Reply by gloriliale** [reply]
Hi @dremin, feel free to [follow this ticket](https://issues.corp.twilio.com/browse/DSYS-3629) for progress. Many thanks!

**Comment by SiTaggart** [comment]
Hi @dremin 

We have a very simple solution in place right now. On the upgrade guide page, each major version has a link to the last docs build for the latest minor in that major version.

Hopefully that will help elevate some issues with being stuck on older versions of Paste.

https://paste.twilio.design/core/upgrade-guide

**Reply by dremin** [reply]
Thanks! I'm getting a 404 when attempting the doc snapshot link for Paste 10.x. Am I too soon? :)

**Reply by SiTaggart** [reply]
Looks like that's just a bug. You can find v10.20 https://6201b7fc1c305e0007f5619d--paste-docs.netlify.app/

**Reply by dremin** [reply]
It looks like the documentation snapshots have stopped being generated for the newer versions after 16.x (or, at least, they don't seem to have links).

**Reply by SiTaggart** [reply]
Yeah, we're working on a better solution as the preview builds from Netlify are expiring for older versions and disappearing. I don't have an ETA right now, though.
